Scripts/DatabankLib/databankLibrary.py

```diff
@@ -28,7 +28,7 @@
 
 logger = logging.getLogger("__name__")
 
-from databank_defs import *
+from .databank_defs import *
```

Comment:
to be internally consistent with the imports, could you please define the imported methods explicitly, rather than with * ?

Reply:
done during style alignment.